plugins/eslint-plugin-liferay/docs/rules/group-imports.md

```diff
@@ -0,0 +1,48 @@
+# Group `import` statements and `require` calls (group-imports)
+
+This rule enforces (and autofixes) that `import` statements and `require` calls &mdash; henceforth referred to as just "imports" &mdash; appear in the groups described in [our guidelines](https://github.com/liferay/liferay-frontend-guidelines/issues/60).
```

Comment:
Should probably land that guidelines and then update this link to point to the final decision... which will likely point back here, yeah? 😂

Reply:
Yeah, 🐔  and 🥚  — all the places where I link to that issue are basically placeholders.

Reply:
Updated links to point at: https://github.com/liferay/liferay-frontend-guidelines/blob/master/general/imports.md